subsys/net_core_monitor/Kconfig

```
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NET_CORE_MONITOR
+	bool "NCM (Network Core Monitor) module [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	depends on (SOC_NRF5340_CPUAPP || SOC_NRF5340_CPUNET)
+	help
+	  Enable the Network Core Monitor module.
+
+if NET_CORE_MONITOR
+
+menu "Net Core Monitor"
+
+config NCM_APP_FEEDING_INTERVAL_MSEC
+	default 500
+	int "Application core feeding interval"
```

```suggestion
	int "Application core feeding interval (in ms)"
```
This would be great to note the time unit.